src/wrld.js

```diff
@@ -110,40 +112,45 @@ const Wrld = {
 
     return map.leafletMap;
   },
+  getMapById: (mapId) => _mapObjects[mapId],
 
-  Marker: marker.Marker,
-  marker: marker.marker,
+  // shims & overrides
   Popup: popup.Popup,
   popup: popup.popup,
-  Polygon: polygon.Polygon,
-  polygon: polygon.polygon,
-  Polyline: polyline.Polyline,
-  polyline: polyline.polyline,
+  Circle: circle.Circle,
+  circle: circle.circle,
+  Marker: marker.Marker,
+  marker: marker.marker,
+  Polygon: polygonShim.PolygonShim,
+  polygon: polygonShim.polygonShim,
+  Polyline: polylineShim.PolylineShim,
+  polyline: polylineShim.polylineShim,
+  Rectangle: rectangleShim.RectangleShim,
+  rectangle: rectangleShim.rectangleShim,
+
+  // additions
   Prop: prop.Prop,
   prop: prop.prop,
   Heatmap: heatmap.Heatmap,
   heatmap: heatmap.heatmap,
 
+  // new namespaces
   indoors: indoors,
   space: space,
   themes: themes,
   buildings: buildings,
   indoorMapEntities: indoorMapEntities,
   indoorMapFloorOutlines: indoorMapFloorOutlines,
-
-  getMapById: (mapId) => _mapObjects[mapId]
+  native: {
+    Polygon: polygon.Polygon,
+    polygon: polygon.polygon,
+    Polyline: polyline.Polyline,
+    polyline: polyline.polyline,
+  },
 };
 
-L.popup = popup.popup;
-L.circle = circle.circle;
-L.marker = marker.marker;
-L.polygon = polygonShim.polygonShim;
-L.polyline = polylineShim.polylineShim;
-L.rectangle = rectangleShim.rectangleShim;
-
-window.L = L;
-L.Wrld = Wrld;
-L.eeGeo = L.Wrld;
+// For compatibility with eeGeoWebGL we need L.Wrld present
+window.L["Wrld"] = Wrld;
```

Comment:
Since we now have leaflet's L with Wrld added to it, a common error when porting from wrld.js v0.1 to v1.0 will be keeping code like L.marker().addTo(map) where map is a Wrld.map - this will now fail in undefined ways since L.marker is a Leaflet marker.

Is there any sensible way we could add a check in Wrld.map.addLayer() that the object that is being added is a Wrld object, and not from Leaflet - and if so throw an error.  Or at least will TS complain about a type mismatch at some point?